### DISCUSSION TITLE
关于dom 描述后的渲染问题

### DISCUSSION THREAD

**Discussion by webvs2** [preactjs/preact#4007] [Q&A] [answered]
我很开心官方对我之前问题的积极回复，我想可以转到讨论区来讨论：
我对一个dom进行了描述，但是，我是否能不经过render来直接返回一个真实dom呢？我虽然理解vnode才是整体的精髓。但是这种边界情况有意义，就像我的组件一样，我希望生成一个dom,放在root dom上（当一个Functional Components执行的时候，或者Class Components来new 的时候），同时保持整体出口只有一个render。
我的组件：https://github.com/webvs2/preact-nanometerUI/tree/main/src/plugin/message

**Comment by JoviDeCroock** [comment, marked as answer]
You could try checking out [Preact-render-to-string](https://github.com/preactjs/preact-render-to-string) however I don't think there's much other things to do. You can call `render` from the component and use as-is.

**Reply by webvs2** [reply]
> 您可以尝试查看[Preact-render-to-string](https://github.com/preactjs/preact-render-to-string)，但我认为没有太多其他事情要做。您可以`render`从组件调用并按原样使用。

十分感谢，我在其中找到自己需要的东西，我将尝试

**Comment by webvs2** [comment]
@JoviDeCroock Is there a discord way to get in touch, hoping to get some communication 😃

**Reply by rschristian** [reply]
We use Slack: https://chat.preactjs.com/
